### DISCUSSION TITLE
Is there a way to do a hammer on / pull off over a bar line?

### DISCUSSION THREAD

**Discussion by DavidForss** [CoderLine/alphaTab#786] [Q&A] [answered]
Hi!
I´m trying to do a hammer on over a bar line, but as soon as I put the bar line between the notes, the hammer on dissapears.

 Example without bar line, working as expected:
<img width="148" alt="Screenshot 2022-03-11 at 08 46 31" src="https://user-images.githubusercontent.com/3233721/157825264-21fa1a2e-888e-4f0e-89d6-4ae884ef6e1a.png">

( alphaTex: `3.3{h} 4.3` ) 

And example with bar line, the hammer on dissapears:
<img width="164" alt="Screenshot 2022-03-11 at 08 47 02" src="https://user-images.githubusercontent.com/3233721/157825510-f4ca2d26-799c-4711-bb9e-6508b7372111.png">

( alphaTex: `3.3{h} | 4.3` ) 

Any help appreciated!
 / David

**Comment by DavidForss** [comment]
Any info on this one?
I should also add that it gives the same result in the tab as in the standard notation:

<img width="168" alt="Screenshot 2022-03-15 at 08 09 46" src="https://user-images.githubusercontent.com/3233721/158325675-649b6474-56a9-497e-826c-e872f16b3ebe.png">

( alphaTex: `3.3{h} | 4.3` )

Thanks / David

**Comment by Danielku15** [comment]
Sorry for the delay. Seems this message got lost on my side somehow 😟 . Feels like a bug to me. Your syntax is correct and your expectation too. Hammer on should work across bars.

**Comment by Danielku15** [comment, marked as answer]
I just double checked and it seems in 1.3.0 this bug is already fixed:
![image](https://user-images.githubusercontent.com/674916/158690838-4657e69b-359f-4ed1-b3a1-2dcff4a5e267.png)

**Reply by DavidForss** [reply]
Ok, thats great!
In my project I am actually pulling in the js-files from jsdelivr, but it seems that they are serving v. 1.2.2

https://cdn.jsdelivr.net/npm/@coderline/alphatab@latest/dist/alphaTab.js

I thought that by using @latest, it would be serving the latest version.
Is there any way to get hold of the 1.3.0 version through a cdn somewhere?

Thanks!

**Reply by Danielku15** [reply]
1.3.0 is not yet released. it is currently in development. I don't recommend to use it yet in production use but also the 1.3.0 versions are shipped through JsDelivr. `@latest` points to the latest released version while `@alpha` points to the current development version. 

JsDelivr is a CDN on top of NPM, you can see all available versions and tags here: https://www.npmjs.com/package/@coderline/alphatab?activeTab=versions

**Reply by DavidForss** [reply]
Thanks for the quick reply!
Aha, that makes sense, I'll give `@alpha` a try then!

Btw, thanks for all your hard work on this cool library!
 / David
